integration_test/gce/gce_testing.go

```diff
@@ -748,6 +748,14 @@ func getGcloudConfigDir(ctx context.Context) (string, error) {
 	return out.Stdout, nil
 }
 
+// getDirectoryWithTrailingDot returns the given directory with
+// a trailing '/.'.
+func getDirectoryWithTrailingDot(directory string) string {
+	// Can't just use filepath.Join(directory, "."), because it eats dot path
+	// segments. See https://stackoverflow.com/questions/51669486/filepath-join-removes-dot/51670536#51670536
```

Comment:
You can make this shorter.
```suggestion
	// segments. See https://stackoverflow.com/a/51670536.
```